proposals/lip-0053.md

```diff
@@ -224,7 +224,25 @@ def verifyValidatorsUpdate(ccu: CCU) -> None:
     if len(newActiveValidators) < 1 or len(newActiveValidators) > MAX_NUM_VALIDATORS:
         raise Exception(f"Invalid validators array. It must have at least 1 element and at most {MAX_NUM_VALIDATORS} elements.")
 
+    totalWeight = 0
+    for validator in newActiveValidators:
+        # The bftWeight property of each element is a positive integer.
+        if validator.bftWeight == 0:
```

Comment:
This check is redundant in principle, but I kept it for symmetry with LIP0043. Let me know if you prefer to remove it

Reply:
I think it is good to have it. Otherwise, there could maybe be the case that the total weight is 0.